query-coordinator/src/main/scala/com/socrata/querycoordinator/Secondary.scala

```diff
@@ -29,7 +29,7 @@ class Secondary(secondaryProvider: ServiceProviderProvider[AuxiliaryData],
   }
 
   def secondaryMirrors(secondaryName: String): List[String] =
-    mirrors.get(secondaryName).map(_.toList).getOrElse(Nil)
+    mirrors.filterKeys(secondaryName.contains).values.headOption.getOrElse(Nil)
```

Comment:
secondaryName isn't `pg`, it's `pg5-c-...`. Therefore we'll make the mirror manifest map primary servers to mirror servers by whether the mirror manifest's key is contained inside secondaryName.